sunpy/map/mapbase.py

```diff
@@ -1339,6 +1339,48 @@ def draw_rectangle(self, bottom_left, width, height, axes=None, **kwargs):
 
         return [rect]
 
+    def draw_contours(self, levels=None, percent=True, axes=None, **imshow_args):
+        """
+        Draw contours of the data
+
+        Parameters
+        ----------
+
+        levels : `~numpy.ndarray`, list
+            A list of numbers indicating the level curves to draw;
+            e.g., to draw just the zero contour pass levels=[0]
+
+        percent : bool, default: True
+            The width of the rectangle.
```

Comment:
Should this have a figure hash too?

On Wed, Nov 18, 2015 at 1:55 PM, Stuart Mumford <notifications@github.com>
wrote:

> In sunpy/map/mapbase.py
> <https://github.com/sunpy/sunpy/pull/1614#discussion_r45242552>:
>
> > @@ -1339,6 +1339,48 @@ def draw_rectangle(self, bottom_left, width, height, axes=None, **kwargs):
> >
> >          return [rect]
> >
> > +    def draw_contours(self, levels=None, percent=True, axes=None, **imshow_args):
> > +        """
> > +        Draw contours of the data
> > +
> > +        Parameters
> > +        ----------
> > +
> > +        levels : `~numpy.ndarray`, list
> > +            A list of numbers indicating the level curves to draw;
> > +            e.g., to draw just the zero contour pass levels=[0]
> > +
> > +        percent : bool, default: True
> > +            The width of the rectangle.
>
> This needs updating.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunpy/sunpy/pull/1614/files#r45242552>.
>